src/common/validator.js

```diff
@@ -0,0 +1,24 @@
+import { GAME_SETTING } from './constants.js';
+
+// 자동차 이름 유효성 검사
+const isUniqueCarName = (input) => {
+  const names = input.split(',').map(name => name.trim());
+  return names.length === new Set(names).size;
+};
+
+export const isValidCarName = (input) => {
+  const carNames = input.split(',').map(name => name.trim().toUpperCase());
+  return (
+    carNames.length >= GAME_SETTING.MIN_LENGTH_CAR_NAME &&
+    carNames.every(name => name.length <= GAME_SETTING.MAX_LENGTH_CAR_NAME) &&
+    !carNames.includes('') &&
+    isUniqueCarName(carNames.join(','))
+  );
+};
+
+// 시도 횟수 유효성 검사
+const isNumericTryCount = (input) => /^[1-9]\d*$/.test(input);
+
+export const isValidTryCount = (input) => (
+  input.toString().length >= GAME_SETTING.MIN_TRY_COUNT && isNumericTryCount(input)
```

Comment:
이미 input이 string 타입으로 들어오는 것 같은데 문자열로 변환해주신 이유가 궁금해요!
또한 airbnb에서는 String(something) 으로 형변환하는걸 추천하고있습니다!
https://github.com/airbnb/javascript#coercion--strings

```suggestion
  String(input).length >= GAME_SETTING.MIN_TRY_COUNT && isNumericTryCount(input)
```